src/torchpme/lib/math.py

```diff
@@ -15,39 +14,83 @@ def gamma(x: torch.Tensor) -> torch.Tensor:
 
 
 class CustomExp1(torch.autograd.Function):
```

Comment:
Should we also make this private? I think everybody will use the function, right?